main/src/library/App.flix

```diff
@@ -16,12 +16,13 @@
 mod App {
 
     ///
-    /// Runs the given function `f` handling the `Clock`, `Console`, and `Logger` effects.
+    /// Runs the given function `f` handling the `Clock`, `Console`, `FilePath` and `Logger` effects.
     ///
-    pub def run(f: Unit -> a \ ef): a \ (ef - {Clock, Console, Logger}) + IO =
+    pub def run(f: Unit -> a \ ef): a \ (ef - {Clock, Console, FilePath, Logger}) + IO =
         let r = f |>
             Clock.handle |>
             Console.handle |>
+            FilePath.handle |>
```

Comment:
We can worry about this later, I would lave it out for now. It will cause issues with the Result return type.